NGitLab/Impl/UserClient.cs

```diff
@@ -110,7 +110,7 @@ public void Deactivate(int userId)
     public GitLabCollectionResponse<LastActivityDate> GetLastActivityDatesAsync(DateTimeOffset? from = null)
     {
         var url = "/user/activities";
-        if (from is not null)
+        if (from != null)
```

Comment:
Please keep the pattern matching condition, as it's more readable and _(though unnecessary here)_ `the compiler guarantees that no user-overloaded == or != operator is invoked` according to [Microsoft's documentation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/is).

Reply:
[0cf0975](https://github.com/ubisoft/NGitLab/pull/682/commits/0cf09752113b6c122425856f515b152a978a7e83)